Solutions/Digital Shadows/Analytic Rules/Digital_Shadows_incident_creation_exclude.yaml

```diff
@@ -0,0 +1,80 @@
+﻿
+id: f7abe9c1-1e6c-4317-b907-25769e7764c5
+name: Digital Shadows incident creation rule app 1
+kind: Scheduled
+properties:
+  displayName: Digital Shadows Incident rule - App 1
```

Comment:
both rules have the same `name` and `displayName`. Can we change them as I suggested?

Reply:
see:

- https://github.com/digitalshadows/Azure-Sentinel/pull/2#discussion_r836412844
- https://github.com/digitalshadows/Azure-Sentinel/pull/2#discussion_r836413066
- https://github.com/digitalshadows/Azure-Sentinel/pull/2#discussion_r836413482
- https://github.com/digitalshadows/Azure-Sentinel/pull/2#discussion_r836413847

Reply:
Changed names as
-Digital Shadows Incident Creation for include-app
-Digital Shadows Incident Creation for exclude-app